### ISSUE TITLE
结构体带json tag的情况下代码生成错误

### ISSUE THREAD

**Issue by haohanyuzmx** [MashiroC/begonia#15]
```go
type HHH struct {
   Aint    int    `json:"aint"`
   Astring string `json:"astring"`
}//原结构体
type HHH struct {
   Aint    int    `json:&#34;aint&#34;`
   Astring string `json:&#34;astring&#34;`
}//代码生成结果
```

**Comment by MashiroC** [comment]
closed by #16

**Comment by MashiroC** [comment]
closed by #16
